docs/references/subcommands/snippets/commit.md

```diff
@@ -0,0 +1,43 @@
+## `fossa snippets commit`
+
+This subcommand commits the analysis performed in the `analyze` subcommand into a `fossa-deps` file.
```

Comment:
Should we link to the [fossa-deps] docs here? https://github.com/fossas/fossa-cli/blob/master/docs/references/files/fossa-deps.md

Reply:
Yep, updated!